data/themes/default/style.css

```diff
@@ -114,9 +113,15 @@ PianoRoll {
 	qproperty-gridColor: rgb( 128, 128, 128 );
 	qproperty-noteModeColor: rgb( 255, 255, 255 );
 	qproperty-noteColor: rgb( 119, 199, 216 );
-	qproperty-barColor: #4afd85;
 	qproperty-noteBorderRadiusX: 5;
 	qproperty-noteBorderRadiusY: 2;
+	qproperty-selectedNoteColor: rgb( 0, 64, 192 );
+	qproperty-barColor: #4afd85;
```

Comment:
@zonkmachine they should be the same functionally.  The rgb version can take a 4th parameter, alpha, but in this case @Umcaruje didn't add this but rather moved it down a few lines. :+1: